client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -66,44 +84,59 @@ const ManagePluginsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 			}
 		>
 			{ label }
-		</PopoverMenuItem>
+		</MenuItemLink>
 	);
 };
 
 const HostingConfigItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 	const { __ } = useI18n();
 
 	return (
-		<PopoverMenuItem
+		<MenuItemLink
 			href={ getHostingConfigUrl( site.slug ) }
 			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_hosting_config_click' ) }
 		>
 			{ __( 'Hosting configuration' ) }
-		</PopoverMenuItem>
+		</MenuItemLink>
 	);
 };
 
-const alignExternalIcon = css`
-	.gridicons-external {
-		top: 0px;
-	}
-`;
+const ExternalGridIcon = styled( Gridicon )( {
+	top: '-1px',
+	marginLeft: '4px',
+	position: 'relative',
+} );
 
 const WpAdminItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 	const { __ } = useI18n();
 
 	return (
-		<PopoverMenuItem
-			className={ alignExternalIcon }
+		<MenuItemLink
 			href={ site.options?.admin_url }
-			isExternalLink
 			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_wpadmin_click' ) }
 		>
-			{ __( 'Visit WP Admin' ) }
-		</PopoverMenuItem>
+			{ __( 'Visit WP Admin' ) } <ExternalGridIcon icon="external" size={ 18 } />
+		</MenuItemLink>
 	);
 };
 
+const SiteMenuGroup = styled( MenuGroup )( {
+	'> div[role="group"]': {
+		display: 'flex',
+		flexDirection: 'column',
+	},
+} );
+
+const SiteDropdownMenu = styled( DropdownMenu )( {
+	'> .components-button': {
+		padding: '0px',
+		minWidth: '0px',
```

Comment:
```suggestion
		padding: 0,
		minWidth: 0,
```

Let's remove the units here: https://stackoverflow.com/questions/7923042/when-specifying-a-0-value-in-css-should-i-explicitly-mark-the-units-or-omit